Assignment/Assignment/SampleData.cs

```diff
@@ -1,30 +1,101 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.IO;
+using System.Linq;
 
 namespace Assignment
 {
     public class SampleData : ISampleData
     {
+
+        private string _filePath;
+
+        public SampleData()
+        {
+            _filePath = "People.csv";
+        }
+
+        public SampleData(string filePath)
+        {
+            _filePath = filePath!;
+        }
+
         // 1.
-        public IEnumerable<string> CsvRows => throw new NotImplementedException();
+        public IEnumerable<string> CsvRows
+        {
+            get
+            {
+                return File.ReadAllLines(_filePath).Skip(1);
+            }
+        }
 
         // 2.
-        public IEnumerable<string> GetUniqueSortedListOfStatesGivenCsvRows() 
-            => throw new NotImplementedException();
+        public IEnumerable<string> GetUniqueSortedListOfStatesGivenCsvRows()
+        {
+            IEnumerable<string> stateList = 
+                (
+                    from line in CsvRows
+                    let state = line.Split(',')
+                    orderby state[6]
+                    select state[6]
+                )
+                .Distinct().ToList();

```

Comment:
```suggestion
          return CsvRows
                    .Select(item => item.Split(',')[6])
                    .Distinct()
                    .OrderBy(item => item);
```
It's a bit unexpected to use the query flavor of the LINQ syntax - but if you do I would not recommend mixing the two flavors.

Above is the way to go about it with the method syntax, which is a bit more C# looking and, in my opinion, fits better in your code in terms of readability.

Additionally, you do not need to convert this to a list, as the method returns an `IEnumerable`